awxtesting/mockserver/mockdata/groups.go

```diff
@@ -10,7 +10,7 @@ var (
         "results": [
             {
                 "id": 21,
-                "type": "group",
+                "type": 0,
```

Comment:
I think it should be string type, need to confirm

Reply:
me too